### PR TITLE
Check and reset if grpc client serverID mismatch with session

### PR DESCRIPTION
**Pull request by wayblink**
/kind bug
#26471

### CODE REVIEW

**Comment by wayblink** [comment]
rerun ut

**Comment by wayblink** [comment]
rerun ut

File: internal/util/grpcclient/client.go [hunk 1]
```diff
@@ -294,10 +297,20 @@ func (c *ClientBase[T]) callOnce(ctx context.Context, caller func(client T) (any
 		return ret, nil
 	}
 
-	if !funcutil.CheckCtxValid(ctx) {
```
Comment (bigsheeper) [inline internal/util/grpcclient/client.go:297]: Should not remove this?

**Pull request merged**